WireAnalytics/Sources/WireAnalytics/Events/Segmentation/Predefined/Segmentation+Call.swift

```diff
@@ -0,0 +1,73 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+extension AnalyticsEvent.Segmentation {
+
+    enum Call {
+
+        /// Creates a ``AnalyticsEvent.Segmentation`` for indicating whether any screen sharing happened during the call. (including
+        /// other participants)
+
+        static func wasScreenShared(_ value: Bool) -> AnalyticsEvent.Segmentation {
+            .init(key: "call_screen_share", value: value)
+        }
+
+        /// Creates a `SegmentationEntry` for providing the total time in seconds any screen sharing happened in the
```

Comment:
You might want to do a search and replace of `SegmentationEntry`.

Reply:
Done, thanks!
4cf696d28a2d29bccc5d1a0f07e6ca1624464f15